pkg/virt-api/webhooks/validating-webhook/admitters/migration-create-admitter_test.go

```diff
@@ -188,17 +167,17 @@ var _ = Describe("Validating MigrationCreate Admitter", func() {
 		})
 
 		It("should reject Migration spec on create when VMI is finalized", func() {
-			vmi := api.NewMinimalVMI("testmigratevmi3")
+			vmi := libvmi.New()
 			vmi.Status.Phase = v1.Succeeded
-
-			mockVMIClient.EXPECT().Get(gomock.Any(), vmi.Name, gomock.Any()).Return(vmi, nil)
+			virtClient := kubevirtfake.NewSimpleClientset(vmi)
+			migrationCreateAdmitter := NewMigrationCreateAdmitter(virtClient)
 
 			migration := v1.VirtualMachineInstanceMigration{
 				ObjectMeta: metav1.ObjectMeta{
-					Namespace: "default",
+					Namespace: vmi.Namespace,
```

Comment:
IIRC, `libvmi.New()` returns a VMI with an unset Namespace.
There is an option `libvmi.WithNamespace`.